src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -646,6 +646,16 @@ static Linker nativeLinker() {
         /**
          * {@return a linker option used to denote the index of the first variadic argument layout in the
          *          function descriptor associated with a downcall linkage request}
+         * <p>
+         * The {@code index} value must be greater than zero, and less than or equal to the number of argument layouts
```

Comment:
Why > 0 and not >= 0? If the list of argument layout is empty, wouldn't the first vararg be in position 0? E.g. a call to a function `m(...)` with _no_ actual arguments?

Reply:
E.g. are we making an assumption here that if no arguments are passed, then the calling sequence won't be affected by variadic-ness - is that assumption always true?

Reply:
Ah, sorry, this is just a  mistake I made. It should be >= 0